games-board/cockatrice/cockatrice-20161030.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils
+
+DESCRIPTION="An open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+SRC_URI="${HOMEPAGE}/archive/2016-10-30-Release.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="server +client +oracle"
+
+DEPEND="
+	dev-libs/libgcrypt:0
+	dev-libs/protobuf
+	dev-qt/linguist-tools:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtprintsupport:5
+	client? (
+		dev-qt/qtgui:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtsvg:5
+	)"
+RDEPEND="
+	dev-qt/qtcore:5 "
+
+# As the default help/about display the sha1 we need it
+SHA1='3498b16'
+
+S="${WORKDIR}/Cockatrice-2016-10-30-Release"
+
+src_configure() {
+	local mycmakeargs=(
+		-DWITH_CLIENT=$(usex client 1 0)
```

Comment:
I will try to let the default flag and upgrade the branch this seem's little change to me.